nifi-framework-api/src/main/java/org/apache/nifi/action/FlowActionReporterCreationException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.action;
+
+/**
+ * An exception that will be thrown if a reporter can not be created.
+ */
+public class FlowActionReporterCreationException extends RuntimeException {
+
+    public FlowActionReporterCreationException() {
+    }
+
+    public FlowActionReporterCreationException(String message) {
+        super(message);
+    }
+
+    public FlowActionReporterCreationException(String message, Throwable cause) {
+        super(message, cause);
+    }
```

Comment:
The other two constructors without message arguments should be removed to require some description.